roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/entity/StreetSection.java

```diff
@@ -56,8 +64,11 @@ public void moveFirstCarToNextSection() {
 
     }
 
-    @Override
-    public void addCar(ICar car) {
+    public Queue<Car> getQueue() {
```

Comment:
i dont think it should be possible to manipulate the queue from outside, this is a hidden attribute of the section

Reply:
i think the same, the user should not know what data structure is behind the street section